src/Polar/kernels.jl

```diff
@@ -560,60 +669,100 @@ KA.@kernel function adj_reactive_power_edge_kernel!(
     end
 end
 
+# Refresh active power (needed to evaluate objective)
+function adj_reactive_power!(
```

Comment:
Rename as `adjoint_reactive_power!` ?

Reply:
This will be taken care off in a future naming PR.